### PR TITLE
fix: named event args

### PR DESCRIPTION
**Pull request by ronag**
Use named arguments (object) for events.

### CODE REVIEW

File: docs/api/Dispatcher.md [hunk 1]
```diff
@@ -734,21 +734,25 @@ Returns: `Boolean`, `false` if user should wait for `'drain'` event before calli
...
 
 Parameters:
 
-* **origin** `URL`
-* **targets** `Array<Dispatcher>`
-* **error** `Error`
+* `Object`
+  * **url** `URL`
+  * **targets** `Array<Dispatcher>`
+  * **error** `Error`
```
Comment (ronag) [inline docs/api/Dispatcher.md:748]:
```suggestion
  * **err** `Error`
```

**Review by mcollina — CHANGES_REQUESTED** [review]
I do not understand this change. Multiple event arguments are customary in Node.js

**Comment by ronag** [comment]
> I do not understand this change. Multiple event arguments are customary in Node.js

I got tired trying to figure out what order would make most sense and think it's nice if users don't need to remember the order.

Don't care that much just thought this would be nice. Close?

**Comment by mcollina** [comment]
I would recommend closing.

**Pull request closed without merging**